### PR TITLE
Update Forge and Fabric to 1.19.3

### PR DESCRIPTION
**Pull request by JT122406**

### CODE REVIEW

File: common/build.gradle [hunk 1]
```diff
@@ -48,7 +48,7 @@ dependencies {
         exclude(module: 'guava')
     }
 
-    api 'com.google.code.gson:gson:2.7'
+    api 'com.google.code.gson:gson:2.8.9'
```
Comment (lucko) [inline common/build.gradle:51]: Please revert this

File: fabric/src/main/java/me/lucko/luckperms/fabric/listeners/FabricConnectionListener.java [hunk 2]
```diff
@@ -46,11 +44,12 @@
 import net.minecraft.server.network.ServerLoginNetworkHandler;
 import net.minecraft.server.network.ServerPlayNetworkHandler;
 import net.minecraft.server.network.ServerPlayerEntity;
-import net.minecraft.util.dynamic.DynamicSerializableUuid;
 
 import java.util.UUID;
 import java.util.concurrent.CompletableFuture;
 
+import static net.minecraft.util.Uuids.getUuidFromProfile;
```
Comment (lucko) [inline fabric/src/main/java/me/lucko/luckperms/fabric/listeners/FabricConnectionListener.java:51]: No static imports please

File: fabric/src/main/java/me/lucko/luckperms/fabric/listeners/FabricConnectionListener.java [hunk 3]
```diff
@@ -36,7 +35,6 @@
 import me.lucko.luckperms.fabric.LPFabricPlugin;
 import me.lucko.luckperms.fabric.mixin.ServerLoginNetworkHandlerAccessor;
 import me.lucko.luckperms.fabric.model.MixinUser;
-
```
Comment (lucko) [inline fabric/src/main/java/me/lucko/luckperms/fabric/listeners/FabricConnectionListener.java:39]: Please don't re-organise imports

File: forge/src/main/java/me/lucko/luckperms/forge/context/ForgePlayerCalculator.java [hunk 4]
```diff
@@ -28,14 +28,8 @@
...
 import me.lucko.luckperms.forge.LPForgePlugin;
-
-import net.luckperms.api.context.Context;
-import net.luckperms.api.context.ContextCalculator;
-import net.luckperms.api.context.ContextConsumer;
-import net.luckperms.api.context.ContextSet;
-import net.luckperms.api.context.DefaultContextKeys;
-import net.luckperms.api.context.ImmutableContextSet;
-import net.minecraft.core.Registry;
+import net.luckperms.api.context.*;
```
Comment (lucko) [inline forge/src/main/java/me/lucko/luckperms/forge/context/ForgePlayerCalculator.java:31]: Please don't re-organise imports

File: velocity/build.gradle [hunk 5]
```diff
@@ -1,17 +1,17 @@
...
+    maven { url 'https://repo.papermc.io/repository/maven-public/' }
 }
 
 dependencies {
     implementation project(':common')
 
-    compileOnly 'com.velocitypowered:velocity-api:3.0.0'
-    annotationProcessor 'com.velocitypowered:velocity-api:3.0.0'
+    compileOnly 'com.velocitypowered:velocity-api:3.1.2-SNAPSHOT'
+    annotationProcessor 'com.velocitypowered:velocity-api:3.1.2-SNAPSHOT'
```
Comment (lucko) [inline velocity/build.gradle:14]: Please revert this change

**Review by lucko — CHANGES_REQUESTED** [review]
Hi, thanks for this! Happy to accept the Forge/Fabric changes, but some of the other miscallaneous changes seem unnecessary, and in some cases there's a good reason to leave as-is.

Would you be able to tidy up slightly, then I will merge. :)

File: forge/src/main/java/me/lucko/luckperms/forge/context/ForgePlayerCalculator.java (hunk 4 above)
Comment (JT122406) [reply forge/src/main/java/me/lucko/luckperms/forge/context/ForgePlayerCalculator.java]: Fixed in 05a819d

File: common/build.gradle (hunk 1 above)
Comment (JT122406) [reply common/build.gradle]: Fixed in 05a819d

File: velocity/build.gradle (hunk 5 above)
Comment (JT122406) [reply velocity/build.gradle:14]: You sure you want this reverted as 3.1.2-SNAPSHOT is the current paper version most people use? if you still do, do you want it reverted to 3.0.0 or 3.1.1 as that one is also used more than 3.0.0

File: fabric/src/main/java/me/lucko/luckperms/fabric/listeners/FabricConnectionListener.java (hunk 3 above)
Comment (JT122406) [reply fabric/src/main/java/me/lucko/luckperms/fabric/listeners/FabricConnectionListener.java]: fixed in a547c40

File: fabric/src/main/java/me/lucko/luckperms/fabric/listeners/FabricConnectionListener.java (hunk 2 above)
Comment (JT122406) [reply fabric/src/main/java/me/lucko/luckperms/fabric/listeners/FabricConnectionListener.java]: fixed in a547c40

File: velocity/build.gradle (hunk 5 above)
Comment (lucko) [reply velocity/build.gradle:14]: 3.1.1 is fine :)

Comment (JT122406) [reply velocity/build.gradle]: addressed in 0220aa8 :)

**Review by lucko — APPROVED** [review]

**Comment by lucko** [comment]
Thank you :)
